apps/vercel/frontend/src/locations/ConfigScreen.tsx

```diff
@@ -154,7 +148,14 @@ const ConfigScreen = () => {
     setIsAppConfigurationSaved(false);
   };
 
-  const renderPostAuthComponents = !isAuthenticationError && parameters.vercelAccessToken;
+  const renderPostAuthComponents =
+    !isAuthenticationError && hasTokenBeenValidated && parameters.teamId;
+  const renderPostProjectSelectionComponents =
+    renderPostAuthComponents &&
+    !errors.projectSelection.projectNotFound &&
+    !errors.projectSelection.cannotFetchProjects &&
+    parameters.selectedProject &&
+    projects.length;
```

Comment:
render the API path section when there are no blocking project errors, there is a selected project, and there exist projects at all.